client/signup/types.ts

```diff
@@ -21,7 +21,9 @@ export interface Flow {
 	showRecaptcha?: boolean;
 	enableBranchSteps?: boolean;
 	hideProgressIndicator?: boolean;
-	enablePresales?: boolean;
```

Comment:
There are a few flows that still have `enablePresales` prop. It would be great to delete it because otherwise it will probably stay forever.

Reply:
I checked calypso for `enablePresales` in this PR, but I couldn't find anything left! Have you found something?

Reply:
Yes, I found 3 in `flows-pure.js`

Reply:
You're right. My bad!